Resources/Prototypes/Entities/Clothing/Shoes/specific.yml

```
@@ -283,4 +283,4 @@
     footstepSoundCollection:
       collection: FootstepSkates
       params:
-        variation: 0.08
+        variation: 0.08
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)